static/app/components/replays/canvasReplayerPlugin.tsx

```diff
@@ -173,8 +177,17 @@ export function CanvasReplayerPlugin(events: eventWithTime[]): ReplayPlugin {
    * The image element is saved to `containers` map, which will later get
    * written to when replay is being played.
    */
-  function cloneCanvas(id: number, node: HTMLCanvasElement) {
-    const cloneNode = node.cloneNode() as HTMLCanvasElement;
+  function cloneCanvas(id: number, node: Node) {
+    if (!isCanvasElement(node)) {
+      return null;
+    }
+    const cloneNode = node.cloneNode();
+
+    if (!isCanvasElement(cloneNode)) {
```

Comment:
is this too verbose to check it again?

Reply:
it checks the cloned node, which not sure how it can return a diff element type than `node`, but TS was yelling so I threw in a log stmt to see if this actually happens or not.